common/app/Nav/Nav.jsx

```diff
@@ -204,7 +204,10 @@ export class FCCNav extends React.Component {
               />
             </a>
           </NavbarBrand>
-          <FCCSearchBar dropdown={ true } />
+          <FCCSearchBar
+            dropdown={ true }
+            placeholder='&#xf002; What would you like to know?'
```

Comment:
Consider using a FA icon here instead.

Reply:
Believe me, this is the 'least' hacky way I could get a FA icon in the placeholder, other methods require crazy css placement over the input, and there was never a guarantee that the icon would be where we needed it.

This is no HMTL allowed in a placeholder, unfortunately.